patches/third_party-android_deps-BUILD.gn.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/third_party/android_deps/BUILD.gn b/third_party/android_deps/BUILD.gn
+index 17a0deffa488ba3ff89fc8e87f5a3308afaff790..e5de0dd6b5b99ce809e013c2db149110360a8010 100644
+--- a/third_party/android_deps/BUILD.gn
++++ b/third_party/android_deps/BUILD.gn
+@@ -3,7 +3,7 @@
+ # found in the LICENSE file.
+ 
+ import("//build/config/android/rules.gni")
+-
++android_aar_prebuilt("google_play_services_safetynet_java") { aar_path = "libs/com_google_android_gms_play_services_safetynet/play-services-safetynet-15.0.1.aar"  info_path = "libs/com_google_android_gms_play_services_safetynet/com_google_android_gms_play_services_safetynet.info" deps = [":google_play_services_base_java"]}
```

Comment:
this should go in a brave gn file, there's no reason to put it here because `google_play_services_package` is not needed for this https://github.com/brave/brave-core/pull/3731/files#diff-b698c0699b41c2f99c4e142c487eae29R26

Reply:
done